src/database/migrations/20190729080444-create-commentlikes.js

```diff
@@ -0,0 +1,46 @@
+module.exports = {
+  up: (queryInterface, Sequelize) =>
+    queryInterface.createTable(
+      'CommentLike',
```

Comment:
This is a migration file, so the `tableName` should be plural. Sequelize should normally do this by default. Please make the changes here